File: numbers/decbin.c
#include <stdio.h>
#include <math.h>


int main() {
	int decArr[3] = {0, 0, 0};
	int decArrLength = sizeof(decArr) / sizeof(decArr[0]);
	int i;
	int temp;

	for (i=0; i<decArrLength; i++) {
		printf("input a number ");
		scanf("\n%d", &temp);

		decArr[i] = temp;
	}

	for (i=0; i<decArrLength; i++) {

		temp = decArr[i];
		printf("%d in binary is: ", temp);
		while (temp > 0.9) {
			printf("%d", temp % 2);
			temp = temp / 2;
		}
		printf("\n");
	}


	return 0;
}

File: numbers/decbin.lua
decimal = {0, 0, 0};

for dindex in pairs(decimal) do
	io.write("input a number: ")
	d = io.read("n")
	decimal[dindex] = d
end

for dindex in pairs(decimal) do

	io.write(decimal[dindex] .. " in binary is: ")
	while decimal[dindex] > 0.9 do
		io.write(math.floor(decimal[dindex] % 2))
		decimal[dindex] = decimal[dindex] / 2
	end
	io.write("\n")
end

